client/src/components/Table.jsx

```diff
@@ -79,8 +79,8 @@ const LogTable = props => {
     <>
       <div>
         <TableContainer>
-          <Table className="table">
-            <TableHead className="table-head">
+          <Table stickyHeader>
+            <TableHead>
```

Comment:
the idea was to make the scrollable table that has other functions (like how many rows), I think we will do that in V3 or if I have the time to do it before tomorrow.
The old styles will do now, could you please put the classes back

Reply:
You got it. Ill add those back in.